.github/workflows/build.yml

```diff
@@ -111,6 +111,14 @@ jobs:
       - name: Create environment variable for VM
         id: env-vm-ip
         run: echo "wlsPublicIP=${{steps.query-vm-ip.outputs.publicIP}}" >> $GITHUB_ENV
+      - name: Update applications
+        run: |
+          echo "pubilc IP of VM: ${wlsPublicIP}"
+          echo "yum update starts"
```

Comment:
@majguo I think this line should be moved before line 121

Reply:
@git4rk  it needs to be the last thing that happens before **Deprovision**.